assignments/arrays.js

```diff
@@ -72,25 +78,48 @@ console.log(`Car 33 is a *car year goes here* *car make goes here* *car model go
 let lastCar = 0;
 console.log();
 
+let lastCar = inventory[inventory.length - 1];
+console.log('${lastCar.car_make} ${lastCar.car_model}');
```

Comment:
Remember with a template string you will need to use the back tick ( ` ) instead of the single quotation ( ' ). Check your console.log of this problem, it is printing out ${lastCar.car_make} ${lastCar.car_model}